frontend/src/app/main/ui/ds/storybook.scss

```diff
@@ -1,5 +1,3 @@
-@use "./colors.scss";
```

Comment:
This was causing the `color.scss` to be included twice. Since that stylesheet is considered a CSS module, is automatically added to the end CSS without it having to be included manually.